gulpfile.js

```diff
@@ -125,6 +129,27 @@ gulp.task('generate-index-files', function() {
     .pipe(gulp.dest('./dist'));
 });
 
+gulp.task('beautify', function () {
+  gulp.src('dist/**/*.js')
+      .pipe(beautify({ config: 'node_modules/mofo-style/linters/.jsbeautifyrc' }))
+      .pipe(gulp.dest('dist'));
+});
+
+gulp.task('jshint', function() {
+  return gulp.src('./lib/*.js')
+      .pipe(jshint({ lookup: 'node_modules/mofo-style/linters/.jshintrc' }))
+      .pipe(jshint.reporter('default'));
+});
+
+
+gulp.task('jscs', function () {
+  // jscs doesn't play nice with *.jsx files so we're avoiding lib/*.jsx
+  return gulp.src(['*.js', 'test/*.js', 'test/**/*.js'])
+      .pipe(jscs({ configPath: 'node_modules/mofo-style/linters/.jscsrc' }));
+});
+
+gulp.task('test', ['jscs', 'jshint', 'beautify']);
```

Comment:
I actually added it to the end of `npm test`

but come to think of it they're in the wrong order.